apps/challenges/urls.py

```diff
@@ -6,6 +6,8 @@
 
     url(r'^challenge_host_team/(?P<challenge_host_team_pk>[0-9]+)/challenge$', views.challenge_list,
         name='get_challenge_list'),
+    url(r'^participant_team/(?P<participant_team_pk>[0-9]+)/challenge$', views.challenge_list_participant,
+        name='get_challenge_list_participant'),
```

Comment:
Please change `challenge_list_participant` to `get_participant_team_challenge_list` and `get_challenge_list_participant` to `get_participant_team_challenge_list`.
Always try to create meaningful function names.